docs/reference/indices/split-index.asciidoc

```diff
@@ -4,6 +4,56 @@
 <titleabbrev>Split index</titleabbrev>
 ++++
 
+Splits an existing index into a new index with more primary shards.
+
+[source,console]
+----
+POST /twitter/_split/split-twitter-index
+{
+  "settings": {
+    "index.number_of_shards": 2
+  }
+}
+----
+// TEST[s/^/PUT twitter\n{"settings":{"blocks.write":true}}\n/]
+
+
+[[split-index-api-request]]
+==== {api-request-title}
+
+`POST /<index>/_shrink/<target-index>`
+
+`PUT /<index>/_shrink/<target-index>`
+
+
+[[split-index-api-prereqs]]
+==== {api-prereq-title}
+
+To split an index,
+the index must be marked as read-only
+and have a <<cluster-health,cluster health>> status of `green`.
```

Comment:
This reads as if it's the index that needs to be green, rather than the cluster. If you change the shrink prereqs, I'd make a similar change here:

Before you can split an index:

* The index must be read-only.
* The <<cluster-health, cluster health>> status must be green.

Reply:
Thanks @debadair. I used this format with 6f981b96518da9cef9c3df87e5a546303f42b0f6.